src/tapis-ui/components/apps/AppsListing/AppsListing.tsx

```diff
@@ -1,61 +1,46 @@
 import React from 'react';
-import { NavLink, useRouteMatch } from 'react-router-dom'
+import { useRouteMatch } from 'react-router-dom';
 import { useList } from 'tapis-hooks/apps';
-import { LoadingSpinner, Message, Icon } from 'tapis-ui/_common';
+import { LoadingSpinner, Message } from 'tapis-ui/_common';
 import { Apps } from '@tapis/tapis-typescript';
-import './AppsListing.scss';
+import { Navbar, NavItem } from 'tapis-app/Navbar';
```

Comment:
My hope was that tapis-ui would be lower order components and that tapis-app would be higher order ones. Can Navbar be moved down into tapis-ui if tapis-ui/apps/AppsListing is going to use it?

If it turns out that ultimately any Nav component can never be made generic enough to account for any higher order application of it (because it is intrinsically tied to the router structure of the top level tapis-app) then I'm open to refactoring these nav components into tapis-app, out of tapis-ui.